app/main.py

```diff
@@ -1,6 +1,34 @@
+import requests
+from pprint import pprint
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+URL = "http://api.weatherapi.com/v1/current.json"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    api_key = os.getenv("API_KEY")
+
+    params = {
+        "key": api_key,
+        "q": os.getenv("CITY_NAME")
+    }
+
+    try:
+        response = requests.get(URL, params=params)
+        response.raise_for_status()
+    except requests.exceptions.HTTPError as e:
+        print(e)
+        return
+
+    try:
+        weather_data = response.json()
+    except ValueError as e:
+        print(e)
+        return
```

Comment:
Same recommendation here. Consider logging the exception error message, not just printing it. This will help you keep track of the errors that occur when trying to parse the response as JSON.